### PR TITLE
Set up permissions for actions in plugin

### PR DESCRIPTION
**Pull request by laviro**

### CODE REVIEW

File: lib/foreman_rh_cloud/engine.rb [hunk 1]
```diff
@@ -30,11 +30,19 @@ class Engine < ::Rails::Engine
...
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/reports' => [:last]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/uploads' => [:auto_upload]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/uploads' => [:download_file]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/uploads' => [:last]
+          permission :view_foreman_rh_cloud, :'foreman_rh_cloud/react' => [:inventory_upload]
         end
 
         # Add a new role called 'Discovery' if it doesn't exist
-        role 'ForemanRhCloud', [:view_foreman_rh_cloud]
+        role 'ForemanRhCloud', [:view_foreman_rh_cloud, :generate_foreman_rh_cloud]
```
Comment (ShimShtein) [inline lib/foreman_rh_cloud/engine.rb:43]: Can we also add description for the role? It was proposed in https://github.com/theforeman/foreman_rh_cloud/pull/126

File: lib/foreman_rh_cloud/engine.rb [hunk 2]
```diff
@@ -30,11 +30,19 @@ class Engine < ::Rails::Engine
...
+
+        add_permissions_to_default_roles 'Organization admin' => [:view_foreman_rh_cloud, :generate_foreman_rh_cloud]
```
Comment (ShimShtein) [inline lib/foreman_rh_cloud/engine.rb:45]: I suppose we should add those permissions also to "Manager" and "System admin" roles.

Comment (laviro) [reply lib/foreman_rh_cloud/engine.rb]: Sure, added

File: app/controllers/foreman_inventory_upload/accounts_controller.rb [hunk 3]
```diff
@@ -36,5 +32,9 @@ def status_for(label, job_class)
       label = job_class.output_label(label)
       ForemanInventoryUpload::Async::ProgressOutput.get(label)&.status
     end
+
+    def model_of_controller
+      ::Organization
+    end
```
Comment (ShimShtein) [inline app/controllers/foreman_inventory_upload/accounts_controller.rb:38]:
```suggestion
```

File: app/controllers/foreman_inventory_upload/accounts_controller.rb [hunk 4]
```diff
@@ -1,12 +1,8 @@
 module ForemanInventoryUpload
   class AccountsController < ::ApplicationController
-    # override default "welcome screen behavior, since we don't have a model"
-    def welcome
-      true
-    end
-
     def index
-      labels = Organization.all.pluck(:id, :name)
+      organizations = resource_base
```
Comment (ShimShtein) [inline app/controllers/foreman_inventory_upload/accounts_controller.rb:4]:
```suggestion
      organizations = User.current.my_organizations
```

**Comment by laviro** [comment]
Thanks @ShimShtein for taking care for the PR,
tested and works great with the changes, I think it's ready to be merged

**Comment by ShimShtein** [comment]
Merged, thanks @laviro!

**Pull request merged**